### ISSUE TITLE
Easy-mode production of executable from sdist

### ISSUE THREAD

**Issue by nathanielmanistaatgoogle** [indygreg/PyOxidizer#280]
It would be nice to have a command like `pyoxidizer --build-executable-easy-mode my_module_sdist.tar.gz` that "does the right thing" and produces a single-file executable from the given `.tar.gz`. Taking [this representative sdist](https://github.com/indygreg/PyOxidizer/files/5109459/representative-0.0.20200813144015583612.tar.gz) as an example, the sdist contains a `representative/__main__.py`, so when installed in a Python environment it would be runnable as `python -m representative`, so it makes sense to just build an executable named `representative` for it. There are lot of rich features available when writing a `pyoxidizer.bzl`, but... let me offer the hypotheses that learning to write a `pyoxidizer.bzl` oughtn't strictly be necessary to the simplest use cases and that there's also a lot of value to be found in just transforming sdists to executables.

**Comment by indygreg** [comment]
I agree that this is a reasonable feature request and it should be possible to implement without too much difficulty. This probably exists as a standalone sub-commands with its own minimal options for influencing behavior (target triple, maybe alternate `__main__` function). Under the hood, it would likely create a temporary directory, initialize the `pyoxidizer.bzl` from a template with appropriate settings, run `pyoxidizer build`, and place the output somewhere.
